microprot/scripts/split_search.py

```diff
@@ -577,9 +628,9 @@ def pretty_output(mask_out):
 @click.option('--subseq_fp', '-o', default=None,
               type=click.Path(),
```

Comment:
by default, `click.Path()` has those arguments: `click.Path(exists=False, file_okay=True, dir_okay=True, writable=False, readable=True, resolve_path=False)` and this is what we're going for, hence no need to explicitly define anything else.